### ISSUE TITLE
Noisy logback on startup

### ISSUE THREAD

**Issue by sixinli** [dropwizard/dropwizard#902]
after upgrading to rc-4, there's this log when I start my server:
```
18:51:28,291 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
18:51:28,292 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
18:51:28,292 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.xml]
18:51:28,292 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Setting up default configuration.
18:51:28,357 |-WARN in ch.qos.logback.core.ConsoleAppender[null] - This appender no longer admits a layout as a sub-component, set an encoder instead.
18:51:28,357 |-WARN in ch.qos.logback.core.ConsoleAppender[null] - To ensure compatibility, wrapping your layout in LayoutWrappingEncoder.
18:51:28,357 |-WARN in ch.qos.logback.core.ConsoleAppender[null] - See also http://logback.qos.ch/codes.html#layoutInsteadOfEncoder for details
18:51:29,400 |-INFO in ch.qos.logback.classic.jul.LevelChangePropagator@6dd93a21 - Propagating INFO level on Logger[ROOT] onto the JUL framework
18:51:29,410 |-INFO in c.q.l.core.rolling.TimeBasedRollingPolicy - No compression will be used
18:51:29,412 |-INFO in c.q.l.core.rolling.TimeBasedRollingPolicy - Will use the pattern ./log/server.log for the active file
18:51:29,415 |-INFO in c.q.l.core.rolling.DefaultTimeBasedFileNamingAndTriggeringPolicy - The date pattern is 'yyyy-MM-dd' from file name pattern './log/server.log'.
18:51:29,415 |-INFO in c.q.l.core.rolling.DefaultTimeBasedFileNamingAndTriggeringPolicy - Roll-over at midnight.
18:51:29,417 |-INFO in c.q.l.core.rolling.DefaultTimeBasedFileNamingAndTriggeringPolicy - Setting initial period to Thu Feb 26 18:51:29 EST 2015
18:51:29,419 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - This appender no longer admits a layout as a sub-component, set an encoder instead.
18:51:29,419 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - To ensure compatibility, wrapping your layout in LayoutWrappingEncoder.
18:51:29,419 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - See also http://logback.qos.ch/codes.html#layoutInsteadOfEncoder for details
18:51:29,419 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - File property must be set before any triggeringPolicy or rollingPolicy properties
18:51:29,419 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - Visit http://logback.qos.ch/codes.html#rfa_file_after for more information
18:51:29,419 |-INFO in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - Active log file name: ./log/server.log
18:51:29,419 |-INFO in ch.qos.logback.core.rolling.RollingFileAppender[file-appender] - File property is set to [./log/server.log]
18:51:29,422 |-INFO in ch.qos.logback.classic.AsyncAppender[async-file-appender] - Attaching appender named [file-appender] to AsyncAppender.
18:51:29,422 |-INFO in ch.qos.logback.classic.AsyncAppender[async-file-appender] - Setting discardingThreshold to 51
18:51:29,423 |-WARN in ch.qos.logback.core.ConsoleAppender[console-appender] - This appender no longer admits a layout as a sub-component, set an encoder instead.
18:51:29,423 |-WARN in ch.qos.logback.core.ConsoleAppender[console-appender] - To ensure compatibility, wrapping your layout in LayoutWrappingEncoder.
18:51:29,423 |-WARN in ch.qos.logback.core.ConsoleAppender[console-appender] - See also http://logback.qos.ch/codes.html#layoutInsteadOfEncoder for details
18:51:29,423 |-INFO in ch.qos.logback.classic.AsyncAppender[async-console-appender] - Attaching appender named [console-appender] to AsyncAppender.
18:51:29,423 |-INFO in ch.qos.logback.classic.AsyncAppender[async-console-appender] - Setting discardingThreshold to 51
```

**Comment by arteam** [comment]
Thanks for pointing this. Should be fixed in the last rc6-snapshot.

**Comment by derekcicerone** [comment]
Awesome, thanks for the quick fix!

**Comment by sixinli** [comment]
Thanks for the fix!

**Comment by jeham** [comment]
I still see this noise in 0.8.0.

**Comment by carlo-rtr** [comment]
Indeed. Thanks for reporting. PR is up:
https://github.com/dropwizard/dropwizard/pull/927

On Fri, Mar 6, 2015 at 10:53 AM, Jesper Hammarbäck <notifications@github.com

**Comment by arteam** [comment]
My bad, there were another issue with logback file appender configuration.
Thanks to Carlo for fixing that.

**Comment by sclausen** [comment]
+1 really annoying. can't wait for an update ;)

**Comment by derekcicerone** [comment]
We've also been waiting on a release with a fix. If there is anything we can do to help out to make that happen please let me know.

**Comment by carlo-rtr** [comment]
https://groups.google.com/forum/#!topic/dropwizard-dev/QxAz9PZBA-E
